src/main/java/uk/gov/hmcts/juror/job/execution/jobs/dashboard/ams/data/BureauLettersToBePrintedDB.java

```diff
@@ -14,5 +16,5 @@ public class BureauLettersToBePrintedDB {
     @DatabaseColumn(name = "description", setter = "setDescription")
     public String description;
     @DatabaseColumn(name = "count", setter = "setCount")
-    public Long count;
+    public BigDecimal count;
```

Comment:
Will count ever be a decimal?
If not should this be a Long?

Reply:
Updated